krish98sai/geo-puzzle
# Created puzzle controller and model

Created controller and model for puzzle.

To test:
'git checkout add-puzzle-controller-model'
'docker-compose run web rails db:create'
'docker-compose run web rails db:migrate'
'docker-compose up -d'
Go to the website , make sure it starts up normally.
Make sure there are no errors.
